cudapoa/src/cudapoa_nw_banded.cuh

```diff
@@ -47,31 +47,18 @@ __device__ SizeT get_band_start_for_row(SizeT row_idx, float gradient, SizeT ban
         start_pos = max_column - band_width + CELLS_PER_THREAD;
     };
 
-    start_pos = max(start_pos, static_cast<SizeT>(0));
+    start_pos = max(start_pos, 0);
 
     start_pos = start_pos - (start_pos % CELLS_PER_THREAD);
 
     return start_pos;
 }
 
 template <typename ScoreT, typename SizeT>
-__device__ ScoreT* get_score_ptr(ScoreT* scores, SizeT row, SizeT column, float gradient, SizeT band_width, SizeT max_column)
+__device__ ScoreT* get_score_ptr(ScoreT* scores, SizeT row, SizeT column, SizeT band_start, SizeT band_width)
```

Comment:
is this shared with the adaptive banding file? if so maybe we can move them into a shared `.cuh` file like `banded_nw_utils`?

Reply:
there are some slight differences between the two, but I agree there are multiple kernels similar in banded-NW and adaptive-NW which with some effort and be unified and combined together in a separate file. I'll create an issue to address this in a different PR.

Reply:
sounds good, thanks!